### PR TITLE
Tessa/sidebar wrap

### PR DESCRIPTION
**Pull request by tesk9**
Fixes https://github.com/NoRedInk/noredink-ui/issues/817 (At least partially. The sidebar nav is set up to expand to the full width of its parent container by default. If the /curriculum library is using flexbox and permitting flex grow on the sidebar, then the sidebar may keep growing!)

## Before
the height of each sidebar element was hardcoded to 45px, meaning tons of ugly overflowing content for long text.

## After
the height of each sidebar element is 45px if the content isn't multline, but padding is used to achieve this rather than a hardcoded height value.

cc @NoRedInk/design

### CODE REVIEW

**Comment by bendansby** [comment]
Thank you!

File: src/Nri/Ui/SideNav/V2.elm [hunk 1]
```diff
@@ -240,9 +240,8 @@ viewLockedEntry extraStyles entryConfig =
 
 sharedEntryStyles : List Style
 sharedEntryStyles =
-    [ paddingLeft (px 20)
-    , paddingRight (px 20)
-    , height (px 45)
+    [ padding2 (px 13) (px 20)
+    , Css.property "word-break" "break-word"
```
Comment (lindsaykwardell) [inline src/Nri/Ui/SideNav/V2.elm:244]: Looks like MDN says `word-break: break-word` is deprecated, how do you feel about using the alternate suggestions?

image

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#values

Comment (tesk9) [reply src/Nri/Ui/SideNav/V2.elm]: Great catch, thank you!!! 🙏

**Review by lindsaykwardell — APPROVED** [review]

**Pull request merged**